aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -279,7 +270,7 @@ impl Aggregator {
         {
```

Comment:
Some comments here and below while re-reading these methods for context.

nit: there seem to be some common code that we could extract, e.g. we could have a nice helper for reading & handling the "value_from_storage" (would just provide ::LastCommitted or ::Aggregated as a param)

typo: sideffects -> side effects.

Should we also explain in the comment that the reason this method doesn't return Result<u128> is (I suppose) because LastCommitted read value won't be validated and should not be used other than in predetermined ways?